frontend/src/context/GamemodeContext.jsx

```diff
@@ -35,7 +59,12 @@ export const Reducer = (state, action) => {
 // Initial State
 const initialState = {
     gamemodeType: "Circlefall",
-    gamemodeDataFilePath: "CirclefallNormal.json"
+    gamemodeDataFilePath: "CirclefallNormal.json",
+    gameState: "pregame",
+    resetGame: false,
+    hits: 0,
+    misses: 0,
+    misclicks: 0,
```

Comment:
Good to keep track of, maybe start making issues/tickets for user stuff like this, we would want to eventually save the previous state that the user used maybe (like their last gamemode played?)

Reply:
I agree with making another issue for this. I think things like previous game played should be implemented once we have user data storage created since we would want to keep that information after the user closes the page.